### PR TITLE
Support Celery 5.3 & Python 3.11

### PR DESCRIPTION
**Pull request by stegayet**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -3,6 +3,8 @@ envlist =
     {pypy3,3.7,3.8,3.9}-celery{44,50,51,52,master}-unit,
     # Celery 5.2 added support for Python 3.10.
     3.10-celery{52,53,master}-unit,
+    # Celery 5.3 added support for Python 3.11.
+    3.11-celery{53,master}-unit,
```
Comment (clokep) [inline tox.ini:7]: The matrix here is getting a little out of hand, but I suspect we want to test:

Comment (clokep) [reply tox.ini:7]: I dropped 3.7 in #77 to fix-up CI.

**Comment by clokep** [comment]
I think if you merge in `main` then CI will pass. Alternately you can set the PR to be editable by maintainers.

**Comment by stegayet** [comment]
Thanks for the feedback and fixes, I've rebased onto main, I'll let you have another look :)

**Comment by clokep** [comment]
Thanks! I've published 0.8 with these changes.

**Pull request merged**